scripts/env.sh

```diff
@@ -27,6 +27,11 @@ WORKDIR=/opt/cita-run
 USER_ID=`id -u $USER`
 USER_NAME="user"
 
+if [[ "${1}" =~ "create_cita_config" ]]; then
+    CONTAINER_NAME="cita_config${RELEASE_DIR//\//_}"
+    DOCKER_IMAGE="cita/cita-run:config-ubuntu-18.04-20190228"
```

Comment:
May be `ubuntu-18.04-20190228` can be a variable.
```bash
ID="ubuntu-18.04-20190228"
NAME="cita/cita-run"
if [ ..omit .. ]; then
    VERSION="${ID}"
else
    VERSION="config-${ID}"
fi
DOCKER_IMAGE="${NAME}:${VERSION}"
```

Reply:
get